File: Admin_p/Admin_p.Web/Modules/ServerTypes/electroDB/OrdersForm.ts
import { ServiceLookupEditor, DateEditor, DecimalEditor, StringEditor, PrefixedContext, initFormType } from "@serenity-is/corelib";

export interface OrdersForm {
    UserId: ServiceLookupEditor;
    OrderDate: DateEditor;
    TotalAmount: DecimalEditor;
    Status: StringEditor;
    CreatedAt: DateEditor;
    UpdatedAt: DateEditor;
}

export class OrdersForm extends PrefixedContext {
    static readonly formKey = 'electroDB.Orders';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!OrdersForm.init)  {
            OrdersForm.init = true;

            var w0 = ServiceLookupEditor;
            var w1 = DateEditor;
            var w2 = DecimalEditor;
            var w3 = StringEditor;

            initFormType(OrdersForm, [
                'UserId', w0,
                'OrderDate', w1,
                'TotalAmount', w2,
                'Status', w3,
                'CreatedAt', w1,
                'UpdatedAt', w1
            ]);
        }
    }
}

File: Admin_p/Admin_p.Web/Modules/electroDB/Orders/OrdersForm.cs
using Serenity.ComponentModel;
using System;

namespace Admin_p.electroDB.Forms;

[FormScript("electroDB.Orders")]
[BasedOnRow(typeof(OrdersRow), CheckNames = true)]
public class OrdersForm
{
    public int UserId { get; set; }
    public DateTime OrderDate { get; set; }
    public decimal TotalAmount { get; set; }
    public string Status { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
}